### ISSUE TITLE
Not Support in Blazor WebAssembly

### ISSUE THREAD

**Issue by MarkSky** [zingz0r/CloudFlare.Client#57]
**Describe the bug**
I use this plugin in ASP.Net 6 Blazor WebAssembly, but there is an error: 'Operation is not supported on this platform.'

**Versions**
- CloudfFare.Client NuGet version 6.1.1
- Your project target framework 6.0

**To Reproduce**
1. Install CloudfFare.Client NuGet.
2. Create a button on razor page.
3. Call the Task in the interface.
```
public async Task CleanCDNCache()
{
    try
    {
        string CFEmail = _configuration["CloudflareKey:Email"];

        string CFKey = _configuration["CloudflareKey:AuthKey"];

        string CFZoneID = _configuration["CloudflareKey:ZoneID"];

        using var CFClient = new CloudFlareClient(CFEmail, CFKey);

        await CFClient.Zones.PurgeAllFilesAsync(CFZoneID, true);
    }
    catch (Exception ex)
    {
        Console.WriteLine(ex);
    }
        }
```
4. See the error  in ` using var CFClient = new CloudFlareClient(CFEmail, CFKey);`

**Expected behavior**
Work and no error.

**Comment by zingz0r** [comment]
Yes, not all features of `HttpClientHandler` are supported inside the browser sandbox. 
https://github.com/dotnet/runtime/blob/7f9b8ce5a0d847feab022be1786486bb3432ffd0/src/libraries/System.Net.Http/src/System/Net/Http/BrowserHttpHandler/BrowserHttpHandler.cs#L51-L131

**Comment by zingz0r** [comment]
It will be fixed in the next release (6.1.2)
